### PR TITLE
Add key_inject role made by parity devops team & fix linting

### PR DESCRIPTION
**Pull request by ddorgan**

### CODE REVIEW

File: roles/key_inject/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+      echo "0x$SESSION_KEYS"
+    register: session_keys
+    notify: Restart service
+    when: key_inject_validator_seed is defined
+
+  - name: Show session keys output
+    ansible.builtin.debug:
+      var: session_keys
+    when: key_inject_validator_seed is defined
```
Comment (BulatSaif) [inline roles/key_inject/tasks/main.yml]: `key_inject_validator_seed` was replaced by `key_inject_relay_chain_key_list`. This part of the code was kept for backward compatibility.  We don't need it anymore.

```suggestion
```

File: roles/key_inject/tasks/inject.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+- name: Inject keys
+  block:
+    - name: Inject | Setting {{ item.type }} pub keys
+      ansible.builtin.set_fact:
+        key_inject_pub_key: "{{ (item.priv_key |
+            infrastructure.chain_operations.subkey_inspect(scheme=(item.scheme | default('sr25519')) )).publicKey }}"
```
Comment (BulatSaif) [inline roles/key_inject/tasks/inject.yml]: Plugins is stored [here](https://github.com/paritytech/ansible-galaxy/tree/main/plugins/filter)
```suggestion
           parity.chain.subkey_inspect(scheme=(item.scheme | default('sr25519')) )).publicKey }}"
```

File: roles/key_inject/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+subkey_path: 'https://s3.eu-central-1.amazonaws.com/releases.parity.io/substrate/x86_64-debian%3Astretch/v3.0.0/subkey/subkey'
```
Comment (kogeler) [inline roles/key_inject/defaults/main.yml]:
```suggestion
subkey_path: 'https://releases.parity.io/substrate/x86_64-debian%3Astretch/v3.0.0/subkey/subkey'
```

Comment (BulatSaif) [reply roles/key_inject/defaults/main.yml]: We can just delete it, it is not used (it was used in old version)

**Comment by BulatSaif** [comment]
please fix [ci lint](https://github.com/paritytech/ansible-galaxy/actions/runs/3283624622/jobs/5408578173):
```
./molecule/default/group_vars/all.yml
  Error: 18:1 [empty-lines] too many blank lines (1 > 0)
```

**Review by BulatSaif — APPROVED** [review]

**Pull request merged**